packages/plugin-legacy/README.md

```diff
@@ -173,6 +173,8 @@ These values (without the `sha256-` prefix) can also be retrieved via
 const { cspHashes } = require('@vitejs/plugin-legacy')
 ```
 
+The `regenerator-runtime` polyfill attempts to use `globalThis` object to register itself. If `globalThis` is not available (it is [fairly new](https://caniuse.com/?search=globalThis) and not widely supported, including IE 11), it attempts to perform dynamic `Function(...)` call which violates the CSP. To avoid dynamic `eval` in the absence of `globalThis` consider adding `core-js/proposals/global-this` to `additionalLegacyPolyfills` to define it.
```

Comment:
```suggestion
When using the `regenerator-runtime` polyfill, it will attempt to use the `globalThis` object to register itself. If `globalThis` is not available (it is [fairly new](https://caniuse.com/?search=globalThis) and not widely supported, including IE 11), it attempts to perform dynamic `Function(...)` call which violates the CSP. To avoid dynamic `eval` in the absence of `globalThis` consider adding `core-js/proposals/global-this` to `additionalLegacyPolyfills` to define it.
```

It looks the `regenerator-runtime` is a polyfill added manually by the user, perhaps we can re-phrase that a bit.

Reply:
Thank you for the suggestion